waiter/src/waiter/scheduler/shell.clj

```diff
@@ -686,10 +686,15 @@
     (assoc (get @id->service-agent service-id)
       :syncer (retrieve-syncer-state-fn service-id)))
 
-  (state [_]
-    {:id->service @id->service-agent
-     :port->reservation @port->reservation-atom
-     :syncer (retrieve-syncer-state-fn)})
+  (state [_ include-flags]
+    (cond-> {:supported-include-params ["port-reservation" "services" "syncer"]
+             :type "ShellScheduler"}
+      (contains? include-flags "port-reservation")
+      (assoc :port->reservation @port->reservation-atom)
+      (contains? include-flags "services")
+      (assoc :id->service @id->service-agent)
```

Comment:
I think it's very confusing that the "include" flag doesn't match the key here. (And this isn't the only one.)

Is there a reason you're not just using the key name as the flag name?

Reply:
The key name is very wordy, so used a shorter variant. The include flags are passed in the response so users don't have to guess what flags are supported.

Reply:
Changed the names to match the keys

Reply:
Nick looks like he covered this.